packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -110,6 +110,25 @@ function useStyleWithReset( path, blockName ) {
 	return [ style, setStyle, hasStyle, resetStyle ];
 }
 
+function useFontSizeStyleWithReset(
+	path,
+	blockName,
+	isFluidTypographyEnabled
+) {
+	const [ style, setStyleCallback ] = useStyle( path, blockName );
+	const [ userStyle ] = useStyle( path, blockName, 'user' );
+	const hasStyle = () => !! userStyle;
+	const resetStyle = () => setStyleCallback( undefined );
+	const setStyle = ( newValue, metadata ) => {
+		// Convert font size styles to fluid if fluid is activated.
+		if ( isFluidTypographyEnabled && !! metadata?.slug ) {
+			newValue = `var:preset|font-size|${ metadata?.slug }`;
```

Comment:
Why are we only storing using the `var:` line if fluid typography is enabled? I was wondering if it would also be useful to store the `var:` form for non-fluid typography theme so that we still store a reference to the `var()` rather than the explicit CSS value stored in the preset 🤔

Reply:
That is a good question. I had a problem and I was working backwards 😄 
Makes total sense to use the CSS var whenever a preset is available. I'll update.